thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java

```diff
@@ -374,8 +379,8 @@ public Response createDetectionPipelineApi(
     }
 
     LOG.info("Detection created with id " + detectionConfigId + " using payload " + payload);
-    responseMessage.put("detectionMsg", "Alert was created successfully.");
-    responseMessage.put("detectionMsg-moreInfo", "Record saved with id " + detectionConfigId);
+    responseMessage.put("message", "Alert was created successfully.");
```

Comment:
is there any frontend changes to adapt for this?

Reply:
Frontend uses the toast popup to display this error message.

Earlier, we used to display the messages just below the editor(by parsing detectionMsg and subscriptionMsg) but that seems to be broken. We can clean that up in the frontend and just rely on the toast popup.

Reply:
sounds good. this should work